packages/admin/admin/src/dataGrid/CrudMoreActionsMenu.tsx

```diff
@@ -0,0 +1,193 @@
+import { MoreVertical } from "@comet/admin-icons";
+import { Button, Chip, css, Divider, ListItemIcon, ListItemText, Menu, MenuItem, MenuList, MenuListProps, Typography } from "@mui/material";
+import { Maybe } from "graphql/jsutils/Maybe";
+import * as React from "react";
+import { PropsWithChildren } from "react";
+import { FormattedMessage } from "react-intl";
+
+import { createComponentSlot } from "../helpers/createComponentSlot";
+import { ThemedComponentBaseProps } from "../helpers/ThemedComponentBaseProps";
+
+export type CrudMoreActionsMenuClassKey = "root" | "group" | "divider" | "button" | "chip" | "menuItem";
+
+export interface ActionItem extends React.ComponentProps<typeof MenuItem> {
+    label: React.ReactNode;
+    icon?: React.ReactNode;
+    divider?: boolean;
+}
+
+export interface CrudMoreActionsMenuProps
+    extends ThemedComponentBaseProps<{
+        menu: typeof Menu;
+        menuItem: typeof MenuItem;
+        button: typeof Button;
+        group: typeof CrudMoreActionsGroup;
+        divider: typeof CrudMoreActionsDivider;
+        chip: typeof Chip;
+    }> {
+    selectionSize?: number;
+    overallActions?: Maybe<ActionItem>[];
+    selectiveActions?: Maybe<ActionItem>[];
+}
+
+interface CrudMoreActionsGroupProps {
+    groupTitle: React.ReactNode;
+    menuListProps?: MenuListProps;
+    typographyProps?: React.ComponentProps<typeof Typography>;
+}
+
+function CrudMoreActionsGroup({ groupTitle, children, menuListProps, typographyProps }: PropsWithChildren<CrudMoreActionsGroupProps>) {
+    return (
+        <>
+            <Typography variant="overline" color={(theme) => theme.palette.grey[500]} sx={{ padding: "20px 15px 0 15px" }} {...typographyProps}>
+                {groupTitle}
+            </Typography>
+            <MenuList {...menuListProps}>{children}</MenuList>
+        </>
+    );
+}
+
+const CrudMoreActionsDivider = createComponentSlot(Divider)<CrudMoreActionsMenuClassKey>({
+    componentName: "CrudMoreActionsDivider",
+    slotName: "divider",
```

Comment:
Please make sure the `componentName` is always the name of the overall component instead of having a unique one for each slot. The same goes for the other slots, all slots used in the `CrudMoreActions` component should have `CrudMoreActions` as their `componentName`. 

```suggestion
    componentName: "CrudMoreActions",
    slotName: "divider",
```

Reply:
https://github.com/vivid-planet/comet/pull/2115/commits/32c3a1fb230ca78196f8b022729c56cd7ebc395e